### ISSUE TITLE
debops_filter_plugins.py is duplicated in git, use symlink?

### ISSUE THREAD

**Issue by ypid** [debops/debops#1916]
f4bff00c97cacd46dd2595b657bb3b083dcece90, #1903

Duplicating the files will cause issues when only one of the files gets edited and devs forget to update the other one as well. I am sorry, I cannot fully test this right now so I open this issue instead. Can one of the files be a symlink instead?

**Comment by drybjed** [comment]
No. With conversion to an Ansible Collection, the `ansible_plugns` role doesn't have to exist in the master branch. Older releases will still have it, of course. If somebody wants to use the plugins from the `debops.debops` collection in their "legacy" roles, they will be able to do it by adding that collection in the playbook. Non-module plugins like filters and lookups will need to be referenced using FQCN (so not `parse_kv_items` but `debops.debops.parse_kv_items`) but that's a small cost of updating the references, I think.

We can remove the `ansible_plugins` role and references to it from all DebOps roles that use it. Documentation about the custom plugins should be preserved though, probably in the "User Manual" section.
